### PR TITLE
DOC-690: Document new format_empty_lines option

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: DOC-690, TINY-6483

Description of Changes:
* Document new `format_empty_lines` setting

Pre-checks:
- [X] Branch prefixed with `feature/` or `hotfix/`
- [X] `_data/nav.yml` has been updated (if applicable)
- [X] Files has been included where required (if applicable)
- [X] Files removed have been deleted, not just excluded from the build (if applicable)
- [X] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes56.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ keywords: releasenotes bugfixes
 
 These release notes provide an overview of the changes for {{site.productname}} 5.6, including:
 
-- [TinyMCE 5.6 new features and enhancements](#tinymce55newfeaturesandenhancements)
+- [TinyMCE 5.6 new features and enhancements](#tinymce56newfeaturesandenhancements)
 - [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
 - [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges)
 - [Minor changes for TinyMCE 5.6](#minorchangesfortinymce55)
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:14]:
```suggestion
- [Minor changes for TinyMCE 5.6](#minorchangesfortinymce56)
```

File: _includes/configuration/format-empty-lines.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## `format_empty_lines`
+
+{{ site.requires_5_6v }}
+
+This option allows `inline` formats to be applied to empty lines for multi-line selections.
```
Comment (tylerkelly13) [inline _includes/configuration/format-empty-lines.md:5]:
```suggestion
This option allows "inline" formats to be applied to empty lines for multi-line selections. Inline formats include bold (`<strong>`) and underline (`<span style="text-decoration: underline;">`).
```

File: release-notes/release-notes56.md [hunk 3]
```diff
@@ -24,7 +24,11 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 The following new features and enhancements were added for the {{site.productname}} 5.6 release.
 
-### Improvement Name
+### New `format_empty_lines` option for content formatting
+
+A new `format_empty_lines` option allows empty lines to be formatted for mult-line selections when applying an `inline` format such as `bold`.
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:29]:
```suggestion
A new `format_empty_lines` option allows empty lines to be formatted for mult-line selections when applying an "inline" format such as `bold`.
```

**Review by tylerkelly13 — APPROVED** [review]
LGTM - Few minor suggestions to clarify what inline formats are (I was initially confused)

File: release-notes/release-notes56.md [hunk 4]
```diff
@@ -24,7 +24,11 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 The following new features and enhancements were added for the {{site.productname}} 5.6 release.
 
-### Improvement Name
+### New `format_empty_lines` option for content formatting
+
+A new `format_empty_lines` option allows empty lines to be formatted for mult-line selections when applying an "inline" format such as bold (`<strong>`).
```
Comment (JamesToohey) [inline release-notes/release-notes56.md:29]:
```suggestion
A new `format_empty_lines` option allows empty lines to be formatted for multi-line selections when applying an "inline" format such as bold (`<strong>`).
```

File: _includes/configuration/format-empty-lines.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+## `format_empty_lines`
+
+{{ site.requires_5_6v }}
+
+This option allows "inline" formats to be applied to empty lines for multi-line selections. Inline formats include bold (`<strong>`) and underline (`<span style="text-decoration: underline;">`).
```
Comment (JamesToohey) [inline _includes/configuration/format-empty-lines.md:5]: Are these the only two inline formats that can be applied? If not, is it worth adding the entire set?

**Review by JamesToohey — APPROVED** [review]

File: _includes/configuration/format-empty-lines.md (hunk 5 above)
Comment (JamesToohey) [inline _includes/configuration/format-empty-lines.md:5]:
```suggestion
This option allows "inline" formats to be applied to empty lines for multi-line selections. Exampled of inline formats include bold (`<strong>`) and underline (`<span style="text-decoration: underline;">`).
```

**Review by JamesToohey — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**